internal/kafka/internal/environments/integration.go

```diff
@@ -38,7 +38,6 @@ func (b IntegrationEnvLoader) Defaults() map[string]string {
 		"mas-sso-realm":                     "rhoas",
 		"osd-idp-mas-sso-realm":             "rhoas-kafka-sre",
 		"enable-kafka-external-certificate": "false",
-		"aws-cluster-compute-machine-type":  "m5.xlarge",
```

Comment:
This is an interesting scenario. Notice that for integration and for production env the compute machine type is different.
By moving the config to a config file this is no longer possible in this section.
So with this changes we altered the machine type used in AWS for the integration case.

There's a possible workaround which is using the `ModifyConfiguration` method available there

Reply:
> I think it might not be necessary in this case however as in integration (mocked) we don't really create the machine in real OCM.

Yes, for mocked environment it does matter.

For real environments (when in `development` mode), it'll use the default config in KFM.
For stage/prod the change doesn't matter  as it'll be set via AppInterface: we want to avoid setting defaults for Prod/Stage via this method and use AppInterface for this.

> Do we run real integration tests (non-mocked) with OCM_ENV development or integration?

The nightly tests are run against `development` environment.

Reply:
Then we should perform the change or modify the mocked environment values.

Reply:
I do not think any change is needed. The mocked api server doesn't match against the request body, it only matches against the endpoint e.g https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/4144f5576c6180b73bf5adad8270a2747e1003cb/test/mocks/api_server.go#L494

Reply:
> Yes, for mocked environment it does matter.

Then this does not apply?

Reply:
> > Yes, for mocked environment it does matter.
> 
> Then this does not apply?

Ah that should have been `does not matter`. Will update the comment